### PR TITLE
[6473] Cleaning up main server files, headers, unused functions, etc. (main)

### PR DESCRIPTION
**Pull request by june-releford**

### CODE REVIEW

File: server/core/src/rodsAgent.cpp [hunk 1]
```diff
@@ -1,3 +1,13 @@
+/*
+
+WORKING CHANGES(june):
+
+* removed unused function static void set_agent_process_name(const InformationRequiredToSafelyRenameProcess& info, const int socket_fd)
+
+*/
+
+
+#include "irods/rodsAgent.hpp"
```
Comment (june-releford) [inline server/core/src/rodsAgent.cpp:10]: remove duplicate

File: server/main_server/src/rodsServer.cpp [hunk 2]
```diff
@@ -67,6 +108,7 @@
 #include <fmt/format.h>
 #include <nlohmann/json.hpp>
 
+
```
Comment (june-releford) [inline server/main_server/src/rodsServer.cpp:111]: remove extra line

File: server/main_server/src/rodsServer.cpp [hunk 3]
```diff
@@ -36,10 +70,17 @@
...
+#include "irods/rcMisc.h"
+#include "irods/rodsErrorTable.h"
+#include "irods/getRodsEnv.h"
+#include "irods/procLog.h"
 #include "irods/plugins/api/grid_configuration_types.h"
 #include "irods/get_grid_configuration_value.h"
 #include "irods/set_delay_server_migration_info.h"
 
+
```
Comment (june-releford) [inline server/main_server/src/rodsServer.cpp:83]: remove extra line

File: server/main_server/src/rodsServer.cpp [hunk 4]
```diff
@@ -93,12 +136,15 @@ pid_t agent_spawning_pid{};
...
-uint ServerBootTime;
+// TODO(june): this is set once, then loaded into agents enviromental variables, doesnt need to be global
+unsigned int ServerBootTime;
 int SvrSock;
 
 std::atomic<bool> is_control_plane_accepting_requests = false;
 int failed_delay_server_migration_communication_attempts = 0;
 
+
+
```
Comment (june-releford) [inline server/main_server/src/rodsServer.cpp:147]: remove extra lines

File: server/main_server/src/rodsServer.cpp [hunk 5]
```diff
@@ -521,6 +551,7 @@ namespace
         }
     } // set_delay_server_migration_info
 
+	// TODO(june): why is this an optional bool?
```
Comment (june-releford) [inline server/main_server/src/rodsServer.cpp:554]: explained

File: server/main_server/src/rodsServer.cpp [hunk 6]
```diff
@@ -860,6 +891,155 @@ namespace
         open("/dev/null", O_WRONLY);
         open("/dev/null", O_RDWR);
     } // daemonize
+	
+	void print_usage(char *prog)
```
Comment (june-releford) [inline server/main_server/src/rodsServer.cpp:895]: add const to prog

File: server/main_server/src/rodsServer.cpp [hunk 7]
```diff
@@ -860,6 +891,155 @@ namespace
...
+			}
+			boost::system::error_code err;
+			boost::filesystem::remove( boost::filesystem::path( lockFilePath ), err );
+		}
+		closedir( dirPtr );
+
+		return savedStatus;
+	}
+	
+	// TODO(june): Would this be better as a cron?
```
Comment (june-releford) [inline server/main_server/src/rodsServer.cpp:987]: make this into an issue

File: server/main_server/src/rodsServer.cpp [hunk 8]
```diff
@@ -1147,18 +1324,33 @@ int serverMain(const bool enable_test_mode = false, const bool write_to_stdout =
...
-            return status;
-        }
-
+		// Start read worker thread
+		for (int i = 0; i < NUM_READ_WORKER_THR; ++i) {
+			try {
+				ReadWorkerThread[i] = new boost::thread(readWorkerTask);
+			}
+			catch (const boost::thread_resource_error&) {
+				rodsLog(LOG_ERROR, "boost encountered a thread_resource_error during readWorkerTask thread construction in serverMain.");
```
Comment (june-releford) [inline server/main_server/src/rodsServer.cpp:1333]: go thru all logs, and change serverMain to main

File: server/main_server/src/rodsServer.cpp [hunk 9]
```diff
@@ -1373,6 +1575,21 @@ agentProc_t* getAgentProcByPid(int childPid, agentProc_t** agentProcHead)
...
+int queueConnectedAgentProc(int childPid, agentProc_t* connReq, agentProc_t** agentProcHead)
+{
+    if (!connReq) {
+        return USER__NULL_INPUT_ERR;
+    }
+
+    connReq->pid = childPid;
+
+    boost::unique_lock<boost::mutex> con_agent_lock(ConnectedAgentMutex);
+    queueAgentProc(connReq, agentProcHead, TOP_POS);
```
Comment (june-releford) [inline server/main_server/src/rodsServer.cpp:1587]: investigate if this can fail

**Review by june-releford — COMMENTED** [review]
From team code review

**Comment by korydraughn** [comment]
@june-releford What is the status of this PR?

Also, please mark the comments as resolved if you feel you've taken care of them.

**Pull request closed without merging**